packages/cmr-client/CMRSearchConceptQueue.js

```diff
@@ -33,13 +33,15 @@ class CMRSearchConceptQueue {
    * @param {string} params.format - the result format
    */
   constructor(params = { searchParams: {} }) {
-    this.clientId = params.clientId;
-    this.provider = params.provider;
+    this.clientId = params.cmrSettings.clientId;
```

Comment:
I'm not sure if we have been doing this - but do we need to call this out as a breaking change in the CHANGELOG? Since this function't work now without `params.cmrSettings`

We should add `params.cmrSettings` to the JSdoc

Reply:
Done and done.